RELEASE/scripts/autoscend/quests/level_13.ash

```diff
@@ -1322,24 +1324,79 @@ boolean L13_towerNSTowerBones()
 	uneffect($effect[Psalm of Pointiness]);
 	uneffect($effect[Mayeaugh]);
 	uneffect($effect[Feeling Nervous]);
-	buffMaintain($effect[Tomato Power]);
+	
+	if (my_primestat()!=$stat[mysticality])
+	{
+		auto_equalizeStats(); // uses reagent oil to stabilize stats
+	}
+	
+	// Clear some AT buffs so we have room. Ur-kel is actively harmful since it increases DR
+	uneffect($effect[Ur-Kel's Aria of Annoyance]);
+	uneffect($effect[Polka of Plenty]);
+	uneffect($effect[The Sonata of Sneakiness]);
+	uneffect($effect[Carlweather's Cantata of Confrontation]);
+	uneffect($effect[Ode to Booze]);
+	
+	acquireMP(150, 0);
+	buffMaintain($effect[Jackasses' Symphony of Destruction]);
+	buffMaintain($effect[Stevedave's Shanty of Superiority]);
 	buffMaintain($effect[Seeing Colors]);
 	buffMaintain($effect[Glittering Eyelashes]);
 	buffMaintain($effect[OMG WTF]);
 	buffMaintain($effect[There is a Spoon]);
 	buffMaintain($effect[Song of Sauce]);
 	buffMaintain($effect[Carol of the Hells]);
+	buffMaintain($effect[Sauce Monocle]);
+	buffMaintain($effect[Arched Eyebrow of the Archmage]);
+	buffMaintain($effect[Rosewater Mark]);
+	buffMaintain($effect[Black Eyes]);
+	buffMaintain($effect[Imported Strength]);
+	buffMaintain($effect[Mystically Oiled]);
+	buffMaintain($effect[Tomato Power]);
+	//~ buffMaintain($effect[Visions of the Deep Dark Deeps]);
 	
 	// Maximizer tries to force familiar equipment. and prefers passive dmg a that. Avoid dealing damage from familiar and losing
 	if(canChangeFamiliar())
 	{
-		use_familiar(lookupFamiliarDatafile("gremlins"));		//delevel with no damage. fallback to none if unavailable
+		if (have_familiar($familiar[magic dragonfish]))
+		{
+			use_familiar($familiar[magic dragonfish]); // boosts spell damage
+		}
+		else
+		{
+			use_familiar(lookupFamiliarDatafile("gremlins"));		//delevel with no damage. fallback to none if unavailable
+		}
 		set_property("auto_disableFamiliarChanging", true);
 	}
 	if(my_familiar() != $familiar[none])
 	{
 		addToMaximize("-familiar");
 		equip($slot[familiar], $item[none]);
+		// Try just boosting weight
+		foreach i,it in auto_getListOfNonDamagingFamiliarEquipment()
+		{
+			if (can_equip(my_familiar(),it))
+			{
+				equip($slot[familiar],it);
+				break;
+			}
+		}
+	}
+	
+	if (auto_remainingCandyCaneSlashes()>0)
+	{
+		addToMaximize("+equip "+$item[Candy Cane Sword Cane]);
+	}
+	
+	if (possessEquipment($item[big hot pepper]))
+	{
+		addToMaximize("+equip "+$item[big hot pepper]);
+	}
+	
+	acquireOrPull($item[congressional medal of insanity]);
```

Comment:
petrified wood water purifier (offhand) or wizard's pouch (accessory) are both double lanterns that are worth considering (and easily accessible)